### PR TITLE
[TTAHUB-731] Next Steps Redesign

### PR DESCRIPTION
**Pull request by AdamAdHocTeam**
## Description of change

Redesign for AR Next Steps page.

## How to test

Next Steps should match the new design linked on https://ocio-jira.acf.hhs.gov/browse/TTAHUB-731.

- Can create Save and Submit a report.
- Can Add/Update/Remove steps.
- Failure to populate a step(s) on blur should alert the user.
- Failure to populate a step(s) on submit should alert the user.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-731

### Every PR

- [x] Meets issue criteria
- [x] JIRA ticket status updated
- [x] Code is meaningfully tested
- [x] Meets accessibility standards (WCAG 2.1 Levels A, AA)
~- [ ] API Documentation updated~
~- [ ] Boundary diagram updated~
~- [ ] Logical Data Model updated~
~- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions~

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [x] Update JIRA ticket status

### CODE REVIEW

**Review by jasalisbury — COMMENTED** [review]
Looks good! Two questions and one comment:

* Do we want to prevent adding a new "next step" until the previous "next step" is valid?
* Should we make the text area "auto expand" like some of our other text areas?
* I'm getting a react unique key error in my console

File: frontend/src/pages/ActivityReport/Pages/components/NextStepsRepeater.css [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
+/* Outline a blank step onBlur or form Validation */
+.ttahub-next-steps-repeater .usa-form-group--error .blank-next-step textarea,
+.usa-form-group--error .ttahub-next-steps-repeater .blank-next-step textarea {
+    border: 3px solid #b50909;
+  }
+
+.ttahub-next-steps-repeater textarea {
```
Comment (thewatermethod) [inline frontend/src/pages/ActivityReport/Pages/components/NextStepsRepeater.css:8]: Would remove this rule, as the textarea is set to 100% (which is what I'm seeing) and setting a min-width means that if you squish the screen down you might find the textarea overflowing outside of the screen

Screen Shot 2022-04-28 at 2 17 20 PM

**Comment by thewatermethod** [comment]
Looks good, works as expected! 
+1 to the console warning

**Review by jasalisbury — APPROVED** [review]
:rocket:

File: frontend/src/pages/ActivityReport/Pages/components/NextStepsRepeater.css [hunk 2]
```diff
@@ -0,0 +1,11 @@
+
+.usa-form .ttahub-next-steps-repeater .usa-textarea {
+  /* Keep within bounds of navigator form with or without delete icon */
+  max-width: 460px;
+}
```
Comment (thewatermethod) [inline frontend/src/pages/ActivityReport/Pages/components/NextStepsRepeater.css:5]: not sure, is this actually doing anything? I removed it locally and didn't see any real effect, but maybe I'm not aware of what problem its solving

Comment (AdamAdHocTeam) [reply frontend/src/pages/ActivityReport/Pages/components/NextStepsRepeater.css:5]: If you add two step boxes. Then remove one of the steps. The text box width changes. This keeps it consistent.

**Pull request merged**